### PR TITLE
AWS review feedback

### PR DESCRIPTION
**Pull request by mvcninja**
Fixes/updates from prior [upstream PR](https://github.com/aws/aws-toolkit-vscode/pull/5351)

### CODE REVIEW

File: packages/core/src/docdb/explorer/dbClusterNode.ts [hunk 1]
```diff
@@ -149,7 +149,10 @@ export class DBClusterNode extends DBResourceNode {
     }
 
     override copyEndpoint() {
-        return copyToClipboard(this.cluster.Endpoint!, this.name)
+        if (this.cluster.Endpoint) {
+            return copyToClipboard(this.cluster.Endpoint!, this.name)
```
Comment (kmcginnes) [inline packages/core/src/docdb/explorer/dbClusterNode.ts]: You might not need the `!` any more, now that you are checking for falsy.

**Comment by alexey-temnikov** [comment]
nit: please update the PR description to include the description and how they are tested.

**Review by kmcginnes — APPROVED** [review]

File: packages/core/src/docdb/commands/stopCluster.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import { telemetry } from '../../shared/telemetry'
+import { localize } from '../../shared/utilities/vsCodeUtils'
+import { DBClusterNode } from '../explorer/dbClusterNode'
+
+export function stopCluster(node?: DBClusterNode): Promise<void> {
+    return telemetry.docdb_stopCluster.run(async () => {
+        if (node?.arn && node?.regionCode) {
+            await node.client.stopCluster(node.arn)
+            getLogger().info('docdb:Stop cluster: %O', node.name)
+            void vscode.window.showInformationMessage(
```
Comment (alexey-temnikov) [inline packages/core/src/docdb/commands/stopCluster.ts:17]: Question: Should we display info message before waiting for node.client.stopCluster(node.arn)?

Comment (mvcninja) [reply packages/core/src/docdb/commands/stopCluster.ts:17]: It returns immediately after, doesn't block while stopping

**Pull request merged**